subscriber/api.go

```diff
@@ -0,0 +1,54 @@
+package subscriber
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"go.uber.org/zap"
+
+	"github.com/gorilla/mux"
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/event"
+	"github.com/openfresh/plasma/pubsub"
+)
+
+type Api struct {
+	config  config.Config
+	pubsub  pubsub.PubSuber
+	payload chan event.Payload
+}
+
+func newApi(pb pubsub.PubSuber, errorLogger *zap.Logger, config config.Config) (Subscriber, error) {
```

Comment:
__[Golint]__

func newApi should be newAPI


<!-- Comment by [SideCI](https://sideci.com) -->